sasjs-tests/src/testSuites/Basic.ts

```diff
@@ -47,7 +47,9 @@ export const basicTests = (
         'Should fail on first attempt and should log the user in on second attempt',
       test: async () => {
         await adapter.logOut()
+        await sleep(1000)
```

Comment:
I'm still not convinced about this `sleep`. If the promise is resolving before the logout actually happens, that sounds to me like there's a problem in the actual logout code.

Reply:
It's not only the logout, and with further look seems that Promise returns when the request finishes. But without the delay in between calling the functions to test the multiple logins, it fails. Can't find more details.

Reply:
Shouldn't this now work without the sleep?